sdk/data/azcosmos/go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/Azure/azure-sdk-for-go/sdk/data/azcosmos
 go 1.18
 
 require (
-	github.com/Azure/azure-sdk-for-go v63.2.0+incompatible
+	github.com/Azure/azure-sdk-for-go v67.0.0+incompatible
```

Comment:
What was the reason for this increment?

Reply:
Don't know what happened here. Maybe unnecessary but go mod tidy did something there so thought it might be needed. Might delete it later

Reply:
@jhendrixMSFT do you know if this side effect of `go mod tidy` is expected?

Reply:
Unfortunately yes.  The net change is no impact though.